README.md

```diff
@@ -24,65 +24,24 @@ deployments.
 
 ## Installation for Swarm Users
 
-For installing swarm for using in your environment, use the Docker Swarm
-documentation on [docs.docker.com](http://docs.docker.com/swarm/).
+For instructions on using Swarm in your dev, test or production environment, refer to the Docker Swarm documentation on [docs.docker.com](http://docs.docker.com/swarm/).
 
-## Installation for Swarm Developers
+## Building Swarm from Source
 
-Developers should always download and install from source rather than
-using the Docker image.
-
-### Prerequisites
-
-1. Beginning with Swarm 0.4 golang 1.4.x or later is required for building Swarm. 
-Refer to the [Go installation page](https://golang.org/doc/install#install)
-to download and install the golang 1.4.x or later package.
-> **Note**: On Ubuntu 14.04, the `apt-get` repositories install golang 1.2.1 version by
-> default. So, do not use `apt-get` but install golang 1.4.x manually using the
-> instructions provided on the Go site.
-
-2. Install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
-
-3. Install [godep](https://github.com/tools/godep).
-
-### Clone and build Swarm
-
-> **Note** `GOPATH` should be set when install godep in above step.
-
-Install the `swarm` binary in the `$GOPATH/bin` directory. An easy way to do this 
-is using the `go get` command.
-
-```bash
-$ go get github.com/docker/swarm
-```
-
-You can also do this manually using the following commands:
-
-```bash
-$ mkdir -p $GOPATH/src/github.com/docker/
-$ cd $GOPATH/src/github.com/docker/
-$ git clone https://github.com/docker/swarm
-$ cd swarm
-$ $GOPATH/bin/godep go install .
-```
-
-Then you can find the swarm binary under `$GOPATH/bin`.
-
-From here, you can follow the instructions [in the main documentation](http://docs.docker.com/swarm/),
-replacing `docker run swarm` with just `swarm`.
+To compile Swarm from source code, refer to the instructions in [CONTRIBUTING.md](http://github.com/docker/swarm/blob/master/CONTRIBUTING.md)
 
 ## Participating
 
-You can [contribute](https://github.com/docker/swarm/blob/master/CONTRIBUTING.md) to Docker Swarm in several different ways:
+You can contribute to Docker Swarm in several different ways:
 
   - If you have comments, questions, or want to use your knowledge to help others, come join the conversation on IRC. You can reach us at #docker-swarm on Freenode.
 
   - To report a problem or request a feature, please file an issue.
 
-  - Of course, we welcome pull requests and patches. For information on making feature requests, follow the process suggested [here](https://github.com/docker/docker/blob/master/CONTRIBUTING.md).
+  - Of course, we welcome pull requests and patches. For information on making feature requests, follow the process suggested [here](https://github.com/docker/docker/blob/master/CONTRIBUTING.md).  Setting up a local Swarm development environment is described [here](http://github.com/docker/swarm/blob/master/CONTRIBUTING.md).
```

Comment:
`CONTRIBUTING.md` doesn't really handle feature requests, does it? Plus, the previous bullet says to file an issue. I think that sentence can go away.